massa-models/src/secure_share.rs

```diff
@@ -45,16 +49,17 @@ pub trait Id {
 }
 
 /// Trait that define a structure that can be wrapped.
-pub trait WrappedContent
+pub trait SecureShareContent
 where
     Self: Sized + Display,
 {
-    /// Creates a wrapped version of the object
-    fn new_wrapped<SC: Serializer<Self>, U: Id>(
+    /// Using the provided key-pair, generates values the are used
+    /// by the trust-free data sharing protocols.
+    fn secure<Ser: Serializer<Self>, ID: Id>(
```

Comment:
Maybe rename it to `new_secure`

Reply:
I like it. I'll put that together on Monday.

Reply:
I went with `new_verifiable`. I'm thinking "secure" is a touch redundant to the type name, and it's a chance to document the properties that make it `SecureShare`